sentry_sdk/integrations/flask.py

```diff
@@ -26,6 +28,12 @@
 except ImportError:
     flask_login = None
 
+if flask_login is None:
```

Comment:
Those imports are not necessary when you later use `importlib` anyway. I would like to avoid adding the functionality of providing your own custom module... You can always use `configure_scope()` and friends to add user data if you want to do that. Docs are misssing examples, probably.